klemens_basic_thoughts_1.md

```diff
@@ -0,0 +1,206 @@
+# Design Proposal
```

Comment:
So I'm finally getting back to this in earnest -- sorry for the delay, but slammed at work after return from Aspen.  We have until 6/17 to submit to pre Cologne mailing.